otx/algorithms/common/tasks/training_base.py

```diff
@@ -167,6 +167,14 @@ def template_file_path(self):
         """Model Template file path."""
         return self._task_environment.model_template.model_template_path
 
+    @property
+    def base_data_pipeline_path(self):
```

Comment:
Is this function used?

Reply:
Yes this is called as `self.data_pipeline_path` in the related task.